client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -344,8 +354,17 @@ function recordTaskClickTracksEvent(
 }
 
 // Returns list of tasks/checklist items for a specific flow
```

Comment:
One of the requests for this task is that if a user's email is verified, we do not show this task (rather than show it as complete). That's why the following change to the getArrayOfFilteredTasks() is here.